docs/tutorial/building-a-theme.md

```diff
@@ -0,0 +1,1502 @@
+
+```shell
+cd gatsby-theme-events
+npm publish
+```
+
+> 💡 Because it's namespaced, you'll need to include public access.
+
+Now it's published! After publishing, you'll be able to find your theme on npm at npmjs.com/{yourpackagename}
+
+## Consume a theme in a Gatsby application
+
+Let's test the theme you've created by created!
+
+### Set up a new Gatsby site
+
+Make a new directory called `theme-test`, and set up the project:
+
+```shell
+mkdir theme-test
+cd theme-test
+yarn init -y
+yarn add react react-dom gatsby @jlengstorf/gatsby-theme-events
+```
+
+> 💡 Where it says `@jlengstorf/gatsby-theme-events`, use the theme you just published instead! Or if you didn't want to actually publish your test theme, go ahead and use `@jlengstorf/gatsby-theme-events`.
+
+### Configure the theme
+
+Open up the code for this new project.
+
+Create a new `gatsby-config.js` file in the root:
+
+```javascript:title=theme-test/gatsby-config.js
+module.exports = {
+  plugins: ["@jlengstorf/gatsby-theme-events"],
+}
+```
+
+### Install the Gatsby CLI
+
+Run:
+
+```shell
+yarn global add gatsby-cli
+```
+
+### Run the site
+
+Making sure you're in your `/theme-test` directory, run `gatsby develop` to start the site.
+
+![The new site, running your new Gatsby theme.](./images/building-a-theme-running-theme.png)
+
+### Add some data
+
+In your project, create a directory, `data`. Inside `data`, create a new file, `events.yml`:
+
+```yaml:title=theme-test/data/events.yml
+- name: Party
+  location: My House
+  start_date: 2019-06-26
+  end_date: 2019-06-26
+  url: https://jason.af/party
+```
+
+Save, and you'll see the new event data in your project:
+
+![The new site, with your new test event data.](./images/building-a-theme-add-test-event.png)
+
+## Use component shadowing to override theme components
+
+"Component shadowing" in Gatsby themes allow us to override or modify components in the theme. Let's use our new "theme-test" project to experiment with this.
+
+Start by adding a `src` folder to your project.
+
+Inside `src`, create a folder with the same title as your theme.
+
+> 💡 Note, with a namespaced theme, this will be two directories deep. For example, using the example of `@jlengstorf/gatsby-theme-events`, the file tree of your project will look like this:
+
+```text
+.
+├── data
+│   └── events.yml
+├── src
+│   └── @jlengstorf
+│       └── gatsby-theme-events
+├── gatsby-config.js
+├── package.json
+└── yarn.lock
+```
+
+Anything inside `theme-test/src/@jlengstorf/gatsby-theme-events` will "shadow" the `@jlengstorf/gatsby-theme-events` theme.
+
+### Override theme colors
+
+To test this shadowing, let's override `theme.js`:
+
+```javascript:title=theme-test/src/@jlengstorf/gatsby-theme-events/theme.js
+import { theme as baseTheme } from "@jlengstorf/gatsby-theme-events/src/theme"
+
+export const theme = {
+  ...baseTheme,
+  colors: {
+    ...baseTheme.colors,
+    primary: "blue",
+  },
+}
+```
+
+<!-- @TODO screenshot -->
```

Comment:
Looks like a few screenshots are left. Not sure if you were already tracking these, so putting review comments to call them out.

Reply:
I was tracking these -- I pinged you about a bug encountered while following the tutorial that's preventing this.